schemas/+acq/checkFileversion.m

```diff
@@ -0,0 +1,18 @@
+
+function oldversion=checkFileVersion(fn)
+
+oldversion = true ;
+try
```

Comment:
Can you check it from a mac? i'm using this from my local machine (R2016a but that command was introduced in 2011a)